sympy/combinatorics/graycode.py

```diff
@@ -329,11 +329,17 @@ def gray_to_bin(bin_list):
 
     Examples
     ========
-
+    
     >>> from sympy.combinatorics.graycode import gray_to_bin
     >>> gray_to_bin('100')
     '111'
 
+    example taken from issue #117 from:- https://github.com/sympy/sympy-live/issues/117
```

Comment:
The `test_graycode` file would be a good place for this.

```python
def test_live_issue_117():
    assert bin_to_gray('0100') == '0110'
    assert bin_to_gray('0101') == '0111'
    for bits in ('0100', '0101'):
        assert gray_to_bin(bin_to_gray(bits)) == bits
```

These were confirmed with [online calculator](https://ncalculators.com/digital-computation/binary-gray-code-converter.htm).

I also stopped the tests since you spelled `gray` as `gtay` in the doctest.

Reply:
Thanks for reviewing.
& sorry for that typo.

Reply:
You don't need to put a reference to the issue in the docstring. Putting it in the commit message is sufficient for provenance.